usersexercise/src/main/java/com/Controllers/UsersController.java

```diff
@@ -0,0 +1,72 @@
+package com.Controllers;
+
+import com.Model.Users;
+import com.Services.UserService;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.beans.BeanUtils;
+import com.Model.Response.UserResponse;
+import com.dto.UserDto;
+import com.Model.Request.UserRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("users")
+public class UsersController {
+
+    private final UserService userservice;
+
+    public UsersController(UserService userservice){
+        this.userservice = userservice;
+    }
+
+    @GetMapping(path = "/allUsers")
+    public List<UserResponse> getAllUsers(){
+        List<UserResponse> returnValue = userservice.getAllUsers();
+        return returnValue;
+    }
+
+    @GetMapping
+    public List<UserDto> getUsers(@RequestParam(value = "page", defaultValue = "1") int page,
```

Comment:
OOF! Never return a list of all your users' DTOs! This would have ALL of their information. Very unsafe! Instead, in this method, once you get the list of DTOs back from the service layer, copy them into another list of Response objects